backend/app/tests/services/doctransformer/test_job/conftest.py

```
@@ -68,14 +68,12 @@ def fast_execute_job_func(
 
 
 @pytest.fixture
-def current_user(db: Session, user_api_key: TestAuthContext) -> UserProjectOrg:
+def current_user(db: Session, user_api_key: TestAuthContext) -> AuthContext:
```

Ruff (0.14.8)

Unused function argument: `db`

(ARG001)